x/swaprouter/types/expected_keepers.go

```diff
@@ -1,14 +0,0 @@
-package types
-
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
-// GammKeeper defines the expected interface needed for swaprouter module
-type GammKeeper interface {
-	SwapI
```

Comment:
Oh, do we not need to specify SwapI in GammKeeper within expected_keepers anymore?

Reply:
Gamm keeper implements `SwapI`, it is specified. The `GammKeeper` interface only existed to extend `SwapI`, adding a `GetPoolCreationFee`. This method is now removed so we can remove the interface extension too